packages/components-organizations-registrar/src/PrivateAppRoot.jsx

```diff
@@ -13,31 +13,28 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import React from 'react';
 // react admin
 import { Admin } from 'react-admin';
 
 import PropTypes from 'prop-types';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { QueryClient } from '@tanstack/react-query';
 
-// pages
-
-import React from 'react';
-
 // components
+import MainLayout from './layouts/MainLayout.jsx';
 import Dashboard from './components/Dashboard.jsx';
 import Loading from './components/Loading.jsx';
 import Footer from './components/Footer.jsx';
-import MainLayout from './layouts/MainLayout.jsx';
 import ConsentProvider from './components/ConsentProvider.jsx';
 import initReactAdminAuthProvider from './utils/reactAdminAuthProvider';
 import { useAuth } from './utils/auth/AuthContext';
 import useSignupRedirect from './utils/auth/useSignupRedirect';
-import theme from './theme/theme';
 import remoteDataProvider from './utils/remoteDataProvider';
 import { useConfig } from './utils/ConfigContext';
+import theme from './theme/theme';
 
-export const PrivateAppRoot = ({ children }) => {
+export const PrivateAppRoot = ({ customDataProvider, children }) => {
```

Comment:
what is `customDataProvider` for? Please add a comment to guide a developer

Reply:
@sloops77 customDataProvider - is a routes that our private app wants to add to a dataProvider. For example add all secure message transferring URLs.
From a pr to monorepo:
export const dataResources = {
  SECURE_MESSAGE_TEST: 'test-url',
  SECURE_MESSAGE_SAVE: 'set-url',
  SECURE_MESSAGE_SUPPORTED: 'are-supported',
  SECURE_MESSAGE_TRANSFER: 'transfer-keys',
};